src/screens/NotificationScreen/index.js

```diff
@@ -1,8 +1,48 @@
 import Notification from "../../components/Notification";
+import { createStackNavigator } from '@react-navigation/stack';
+import RadioButton from "../../components/RadioButton";
 
 
-export default function NotificationScreen() {
+const Stack = createStackNavigator();
+
+export default function NotificationScreen({ words, data }) {
     return (
-        <Notification />
+        <Stack.Navigator>
+
+            <Stack.Screen
+                options={{
+                    headerShown: false
+                }}
+                name='Notification'
+            >
+                {(props) => {
+                    return (
+                        <Notification
+                            {...props}
+                            words={words}
+                            data={data}
```

Comment:
Not used in the notification component but still is being sent. Is is intended?

Reply:
Removed from here already